### DISCUSSION TITLE
MCU Fan Not Working Correctly

### DISCUSSION THREAD

**Discussion by NexGen-3D-Printing** [Guilouz/Creality-Helper-Script-Wiki#572] [Issues]
I found the MCU fan would not auto start at 50C, I can turn it on manually and see the temps drop, but if I dont do this, it will just heat up, the display in Mainsail says its running at 100%, but the slider is still set to off, and the fan is not turning.

I edited the fans-control.cfg and commented out the following:

#[output_pin mcu_fan]
#pin: PB2
#pwm: True
#cycle_time: 0.0100
#hardware_pwm: false
#value: 0.00
#scale: 255
#shutdown_value: 0.0

Now its functioning correctly and will turn on and off at the expected temp preset.

**Comment by Guilouz** [comment]
I have removed it.
